packages/react/package.json

```diff
@@ -50,6 +50,7 @@
     "@carbon/telemetry": "0.1.0",
     "classnames": "2.3.2",
     "copy-to-clipboard": "^3.3.1",
+    "debounce": "^1.2.1",
```

Comment:
@francinelucca Could this use the existing `lodash.debounce` that's already installed, or is this package better?

Reply:
done! thankss